apps/judicial-system/backend/src/app/modules/case/limitedAccessCase.service.ts

```diff
@@ -189,9 +189,9 @@ export class LimitedAccessCaseService {
       throw new NotFoundException(`Case ${caseId} does not exist`)
     }
 
-    if (!theCase.seenByDefender) {
+    if (!theCase.openedByDefender) {
```

Comment:
Please consider moving this update. The original issue that lead to the accompanying Asana ticket was that this field was set even if subsequent guard tests failed and the defender was never allowed to see the case. This is still true, but less likely because the `findById` method is no longer called during defender login.

It would probably be safer to set the `openedByDefender` field in the method `getById` in `LimitedAccessCaseController` just before the case is returned.

Reply:
Fixed, could you take another look to make sure I understood correctly 😊?